harpy/_validations.py

```
@@ -237,14 +237,14 @@ def vcf_contig_match(contigs, vcf):
         sys.exit(1)
 
-def validate_popsamples(infiles, popfile, quiet):
+def validate_popsamples(infiles: list[str], popfile: str, quiet: int) -> None:
     """Validate the presence of samples listed in 'populations' to be in the input files"""
     with open(popfile, "r", encoding="utf-8") as f:
         popsamples = [i.split()[0] for i in f.readlines() if i != "\n" and not i.lstrip().startswith("#")]
     in_samples = [Path(i).stem for i in infiles]
     missing_samples = [x for x in popsamples if x not in in_samples]
     overlooked = [x for x in in_samples if x not in popsamples]
-    if len(overlooked) > 0 and not quiet:
+    if len(overlooked) > 0 and not quiet != 2:
```

Ruff (0.8.2)

Use `quiet == 2` instead of `not quiet != 2`

Replace with `==` operator

(SIM202)